src/sentry/south_migrations/0513_auto__add_field_eventattachment_group_id.py

```diff
@@ -0,0 +1,1513 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = True
```

Comment:
We can now do this without being flagged dangerous if we change the migration to create the index concurrently. Then it won’t block.

Reply:
Great, thanks